### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by CommandString**
Fixed error messages and error property options for attrib types

### CODE REVIEW

File: src/Mapper.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ protected function doMapping(): object
                     continue;
                 }
 
-                throw new MapperException("Property {$attribute->name} is not nullable");
+                throw new MapperException("Property {$attribute->name} on ".$this->instance::class.' not nullable');
```
Comment (Exanlv) [inline src/Mapper.php]:
```suggestion
                throw new MapperException(implode('', [
                    'Property ',
                    $attribute->name,
                    " on ",
                    $this->instance::class,
                    ' not nullable'
                ]));
```

Comment (xHeaven) [reply src/Mapper.php]: With all due respect, what the everliving fuck?

Comment (CommandString) [reply src/Mapper.php]: Why is this something I've probably done before

Comment (Exanlv) [reply src/Mapper.php]: Why is this something that I actually saw in production once and when I asked a coworker about it, he actually defended it?

Comment (xHeaven) [reply src/Mapper.php]: > Why is this something I've probably done before

Because you have no soul, heartless code abuser. :(

Comment (xHeaven) [reply src/Mapper.php]:
```suggestion
                throw new MapperException("Property {$attribute->name} on {$this->reflection->name} not nullable");
```

Comment (Exanlv) [reply src/Mapper.php]:
```suggestion
                throw new MapperException(
                    (function ($template, $variables) {
                        $start = 0;
                        while (($start = strpos($template, "{", $start)) !== false) {
                            $end = strpos($template, "}", $start + 1);
                            if ($end === false) {
                                break;
                            }

                            $variableName = substr($template, $start + 1, $end - $start - 1);
                            if (array_key_exists($variableName, $variables)) {
                                $template = str_replace("{" . $variableName . "}", $variables[$variableName], $template);
                            }

                            $start = $end + 1;
                        }

                        return $template;
                    })('Property {aName} on {cName} not nullable', array(
                        'aName' => $attribute->name,
                        'cName' => $this->instance::class
                    ))
                );
```

Comment (CommandString) [reply src/Mapper.php]: Okay I can at least say I've never done this before...

**Review by xHeaven — APPROVED** [review]
LGTM

**Pull request merged**